Commons/src/main/java/eu/neclab/ngsildbroker/commons/datatypes/results/NGSILDOperationResult.java

```diff
@@ -0,0 +1,177 @@
+package eu.neclab.ngsildbroker.commons.datatypes.results;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Set;
+import java.util.Map.Entry;
+
+import com.github.jsonldjava.core.Context;
+import com.google.common.collect.Lists;
+import com.google.common.collect.Maps;
+import com.google.common.collect.Sets;
+
```

Comment:
Remove space and fix indentation in all files..

Reply:
done